recipes/seasocks/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanException, ConanInvalidConfiguration
+import os
+import re
+import shutil
+import textwrap
+
+
+class SeasocksConan(ConanFile):
+    name = "seasocks"
+
+    description = "A tiny embeddable C++ HTTP and WebSocket server for Linux"
+    topics = ("seasocks", "embeddable", "webserver", "websockets")
+    homepage = "https://github.com/mattgodbolt/seasocks"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-2-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_zlib": [True, False],
+    }
+    default_options = {
+        "shared": True,
```

Comment:
```suggestion
        "shared": False,
```

Reply:
Why change this default?

Reply:
The default of 99% of the packages is static.
There are only a few exceptions.

Reply:
Oh! Just seen this! Perfect: thanks!